scripts/defaults.sh

```diff
@@ -6,8 +6,8 @@
 #
 # CAUTION: the sed expression we currently use does not like white-space in the strings
 
-source network-device-names.cfg
-export KSTEST_URL='http://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Everything/$basearch/os/'
+source ./network-device-names.cfg
+export KSTEST_URL='http://dl.fedoraproject.org/pub/fedora/linux/development/rawhide/Everything/x86_64/os/'
```

Comment:
It might be useful when used in kickstart for multiarch testing (which we don't do now anyway and would have to be handled on higher level I guess), but it does not work for the download script. I think we can go with static architecture here. It would maybe stop testing $basearch substitution of the kickstart command implicitly but I don't think it is a big deal.
But I'd use it for all the variables for consistency.

Reply:
Ack, will change all of them. I noticed that `$basearch` is used in other `defaults-*.sh` files as well but not consistently.

Reply:
> Ack, will change all of them. I noticed that `$basearch` is used in other `defaults-*.sh` files as well but not consistently.

Yes, thank you, in other (rhel) files the FTP value is not actually used and this should be cleaned up somehow (in a separate PR).